Resources.html

```diff
@@ -55,7 +57,29 @@
       }
       .row.content {height:auto;} 
     }
+
+    .div-highlight {
+        border-radius: 5px;
+        width: 50%;
+        background: #FFC627;
+    }
+    p {
+        color: black;
+        }
+    b {
+        color: black;
+    }
+    
+    .center {
+        display: flex;
+        justify-content: center; 
+    }
+
+    .row{
+        padding: 50px;
```

Comment:
We should want a lower `padding-top` and `padding-bottom` since that would look much better imo but that is part of the stylistic design of the website and not functional so will be addressed in the future at a later stage of development.